sdk/javascript/src/BaseValue.js

```diff
@@ -19,7 +19,7 @@ const check = (byteSize, value, isSigned) => {
 	}
 
 	if (value < lowerBound || value > upperBound)
-		throw RangeError(`"value" (${value}) is outside of valid ${8 * byteSize}-bit range`);
+		throw RangeError(`"value" (${value}) is outside of valid ${8 * byteSize}-bit range ${lowerBound}`);
```

Comment:
do you mean to print range instead of just lower bound ?

Reply:
didn't mean to  add that at all 😅